lib/topology_docker/node.py

```diff
@@ -67,20 +67,32 @@ class DockerNode(CommonNode):
      of the container inside this directory.
     :param str shared_dir_mount: Mount point of the shared directory in the
      container.
+    :param list|dict environment: Environment variables to pass to the
```

Comment:
@carlos-jenkins if you know how to pass more than one type to Sphinx, please let me know, I haven't found how.